firebase-firestore/src/main/java/com/google/firebase/firestore/Query.java

```diff
@@ -900,6 +899,47 @@ private ListenerRegistration addSnapshotListenerInternal(
         firestore.getClient(), queryListener, activity, wrappedListener);
   }
 
+  @PublicApi
+  public String getPath() {
```

Comment:
Note to self: We need to figure out how we want to expose Collection Group queries (https://github.com/firebase/firebase-android-sdk/pull/233) in general.